optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/AbstractScoreHolder.java

```diff
@@ -114,15 +115,16 @@ public void configureConstraintWeight(Rule rule, Score_ constraintWeight) {
      * Requires @{@link OptaPlannerRuleEventListener} to be added as event listener on {@link KieSession}, otherwise the
      * score changes caused by the constraint matches would not be undone. See
      * {@link DroolsScoreDirector#resetKieSession()} for an example.
+     *
      * @param kcontext The rule for which to register the match.
      * @param constraintUndoListener The operation to run to undo the match.
      * @param scoreSupplier The score change to be undone when constraint justification enabled.
      */
     protected void registerConstraintMatch(RuleContext kcontext,
             final Runnable constraintUndoListener, Supplier<Score_> scoreSupplier) {
         AgendaItem<?> agendaItem = (AgendaItem) kcontext.getMatch();
-        ConstraintActivationUnMatchListener constraintActivationUnMatchListener
-                = new ConstraintActivationUnMatchListener(constraintUndoListener);
+        ConstraintActivationUnMatchListener constraintActivationUnMatchListener = new ConstraintActivationUnMatchListener(
```

Comment:
The style allows lines longer than 120 chars -> GitHub diff windows with scrollbars.